### PR TITLE
[Php74] Apply PHP 7.4 syntax and typed properties (for private modifier)

### PR DESCRIPTION
**Pull request by samsonasik**
|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

[Php74] Apply PHP 7.4 syntax and typed properties (for private modifier) to keep BC.

### CODE REVIEW

File: src/Generator/ClassGenerator.php [hunk 1]
```diff
@@ -446,10 +446,9 @@ public function removeExtentedClass()
      */
     public function setImplementedInterfaces(array $implementedInterfaces)
     {
-        array_map(function ($implementedInterface) {
-            // This loop is just validating that the given `$implementedInterfaces` contains valid syntax/symbols
-            return TypeGenerator::fromTypeString($implementedInterface);
-        }, $implementedInterfaces);
+        // This loop is just validating that the given `$implementedInterfaces` contains valid syntax/symbols
```
Comment (Ocramius) [inline src/Generator/ClassGenerator.php]: I wonder if we can just drop this?

**Review by Ocramius — APPROVED** [review]

File: src/Generator/ClassGenerator.php [hunk 2]
```diff
@@ -446,10 +446,9 @@ public function removeExtentedClass()
...
+        array_map(static fn($implementedInterface) =>
+            TypeGenerator::fromTypeString($implementedInterface), $implementedInterfaces);
```
Comment (samsonasik) [inline src/Generator/ClassGenerator.php]: any way to suppresss this psalm error: `The call to array_map is not used` ? I tried with : 

```bash
@psalm-suppress UnusedFunctionCall
```

but it seems still error, even added to `psalm.xml`:

```xml
        <UnusedFunctionCall>
            <errorLevel type="error">
                <referencedFunction name="UnusedFunctionCall"/>
            </errorLevel>
        </UnusedFunctionCall>
```

File: src/Generator/ClassGenerator.php (hunk 1 above)
Comment (samsonasik) [reply src/Generator/ClassGenerator.php]: I dropped it.

**Comment by samsonasik** [comment]
All green 🎉

**Review by Ocramius — APPROVED** [review]

**Comment by Ocramius** [comment]
Thanks @samsonasik!

**Pull request merged**